docs/blog/2019-05-29-intergrating-graphiql-explorer/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Improvements to GraphiQL IDE - adding GraphiQL Explorer
+date: 2019-05-29
+author: Michal Piechowiak
+tags:
+  - graphql
+  - graphiql-explorer
```

Comment:
```suggestion
 - graphiql-explorer
```

Reply:
seems like prettier demands 2 spaces

Reply:
Something's awry here, then! I'm not even 100% sure that's valid.

Reply:
This follows tags as set in https://raw.githubusercontent.com/gatsbyjs/gatsby/master/docs/blog/2019-05-17-improvements-to-schema-customization/index.md which also use 2 spaces

Reply:
@pieh the tags were showing as indented weirdly--you're right!

![Screen Shot 2019-05-29 at 12 25 01 PM](https://user-images.githubusercontent.com/3924690/58577771-d669ff00-820c-11e9-81e9-a993178cd9b4.png)